Dockerfile

```diff
@@ -20,7 +18,7 @@ RUN mkdir $PROJECT_DIR && \
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
         apt-transport-https build-essential curl git libpq-dev python-dev \
-        default-libmysqlclient-dev gettext sqlite3 libffi-dev libsasl2-dev \
```

Comment:
do you know what libsasl2-dev is used for?

Reply:
for Apache Hive, but we are removing the dependency because it's not used anymore apparently.